### PR TITLE
Fix #52

### LINKED ISSUE #52: [ENHANCEMENT] Confirm discard edit dialog
**Describe the enhancement**
When pressing back or cancel in the `EditMemoryActivity`, a `AlertDialog` should be shown to ask for confirmation if you want to discard your edits.

**Screenshots**
Screenshot 2019-05-09 at 21 05 40

### PR DESCRIPTION
**Pull request by Hicham010**
There is a Discard alertdialog when you try to leave the edit activity.

Fixes #52

### CODE REVIEW

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java [hunk 1]
```diff
@@ -610,10 +610,20 @@ public boolean onSupportNavigateUp() {
 
     @Override
     public void onBackPressed() {
-        super.onBackPressed();
-        if (isEditMode)
-            AnalyticsUtil.cancelEditContent(this);
-        else
-            AnalyticsUtil.cancelAddContent(this);
+        new AlertDialog.Builder(MemoryEditActivity.this)
+                .setTitle("Discard Memory")
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java]: This is not using string resources. Please use them and also do the Dutch translation.

**Review by MrTheGood — CHANGES_REQUESTED** [review]
Also nice work on reformatting the code.

**Review by AnassM06 — APPROVED** [review]
You are the best!

**Review by MrTheGood — APPROVED** [review]
Nice.

**Comment by AnassM06** [comment]
Good job fixing mistakes

**Pull request merged**